src/colvarbias.cpp

```diff
@@ -143,6 +146,37 @@ int colvarbias::init(std::string const &conf)
 }
 
 
+int colvarbias::init_mts(std::string const &conf) {
+  int error_code = COLVARS_OK;
+
+  get_keyval(conf, "timeStepFactor", time_step_factor, time_step_factor);
+
+  if (time_step_factor < 1) {
+    error_code |= cvm::error("Error: timeStepFactor must be 1 or greater.\n", COLVARS_INPUT_ERROR);
+  }
+
+  if (time_step_factor % cvm::proxy->time_step_factor() != 0) {
+    error_code |=
+        cvm::error("timeStepFactor for this bias (currently " + cvm::to_str(time_step_factor) +
+                       ") must be a multiple of the global Colvars timestep multiplier (" +
+                       cvm::to_str(cvm::proxy->time_step_factor()) + ").\n",
+                   COLVARS_INPUT_ERROR);
+  }
+
+  for (auto *cv : colvars) {
+    if (cv->get_time_step_factor() != time_step_factor) {
```

Comment:
I suppose the bias' factor could also be a multiple of of the colvar's, right?

Reply:
Metadynamics or OPES with many kernels can become costly, so MTS would make sense - and then you can have a cheap harmonic restraint that you want at every step.

Reply:
Good point about ABF: for ABF, you can do MTS only if you get forces from the same timestep. The NAMD-style deferred forces scheme will break it, I need to add that condition.

Reply:
Then maybe metadynamics in higher dimension, when you don't want to use grids anymore :-)

Reply:
Frankly speaking, I seldom use the MTS for free-energy calculations. Making the bias time step factor to support a multiple of CV time step factors could be useful if the cost of computing bias is high. OPES doesn't use grids, but it does merge Gaussian kernels and use a neighbor list to reduce the number of hills. For metadynamics, I think @jhenin is right that someone may use metadynamics for more than 4 CVs and avoid the grids, and then use reweighting to estimate the free-energy difference of two states. In such case, the cost of metadynamics could be expensive, and therefore one would resort to MTS.